store/connect.go

```diff
@@ -14,7 +16,13 @@ var (
 	registered = make(map[string]ConnectorFn)
 )
 
-func Connect(ctx context.Context, dsn string) (s Storer, err error) {
+func Connect(ctx context.Context, dsn string, opt options.EnvironmentOpt) (s Storer, err error) {
+	if opt.IsDevelopment() {
+		expr := regexp.MustCompile(`[.\-]+`)
+		version := expr.ReplaceAllString(opt.BuildVersion, "_")
```

Comment:
how about if we just do a quick & dirty `os.Getenv('BUILD_VERSION')` instead of all this option acrobatics?